src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -34,6 +35,7 @@ export const VideoView: React.FC = () => {
   const { state, updateWatchedVideos } = usePersonalData()
   const timestampFromQuery = Number(useRouterQuery('time'))
   const { getAssetUrl } = useAsset()
+  const searchRouteMatch = !!useMatch({ path: absoluteRoutes.viewer.search() })
```

Comment:
Maybe it'd be safer to have the condition the other way around? So to check whether we are on `VideoView`? Then if we add another overlay, this won't break

Reply:
good point!